Wrong signature for GModDataPack on Windows/SRCDS

Using the latest github release build

```
[ERROR] unable to find GModDataPack::SendFileToClient
  1. unknown - [C]:-1
   2. require - [C]:-1
    3. unknown - addons/xmod/lua/includes/sourcenet/server.lua:1
     4. include - [C]:-1
      5. unknown - addons/xmod/lua/includes/sourcenet/outgoing.lua:2
       6. include - [C]:-1
        7. unknown - addons/xmod/lua/includes/sourcenet/gameevents.lua:4
```

```
version
Protocol version 24
Exe version 18.12.05 (garrysmod)
Exe build: 15:58:05 Dec 15 2018 (7343) (4000)
```

--- a/readme.md
+++ b/readme.md
@@ -12,7 +12,7 @@ If stuff starts erroring or fails to work, be sure to check the correct line end
 
 ## Requirements
 
-This project requires [garrysmod\_common][1], a framework to facilitate the creation of compilations files (Visual Studio, make, XCode, etc). Simply set the environment variable '**GARRYSMOD\_COMMON**' or the premake option '**gmcommon**' to the path of your local copy of [garrysmod\_common][1].  
+This project requires [garrysmod\_common][2], a framework to facilitate the creation of compilations files (Visual Studio, make, XCode, etc). Simply set the environment variable '**GARRYSMOD\_COMMON**' or the premake option '**gmcommon**' to the path of your local copy of [garrysmod\_common][2].  
 We also use [SourceSDK2013][3], so set the environment variable '**SOURCE\_SDK**' or the premake option '**sourcesdk**' to the path of your local copy of [SourceSDK2013][3]. The previous links to [SourceSDK2013][3] point to my own fork of VALVe's repo and for good reason: Garry's Mod has lots of backwards incompatible changes to interfaces and it's much smaller, being perfect for automated build systems like Travis-CI.  
 
   [1]: http://christopherthorne.googlecode.com/svn/trunk/gm_sourcenet3

--- a/source/main.cpp
+++ b/source/main.cpp
@@ -136,11 +136,11 @@ namespace global
 	{
 		LUA->CreateTable( );
 
-		LUA->PushString( "sourcenet 1.1.5" );
+		LUA->PushString( "sourcenet 1.1.6" );
 		LUA->SetField( -2, "Version" );
 
 		// version num follows LuaJIT style, xxyyzz
-		LUA->PushNumber( 10105 );
+		LUA->PushNumber( 10106 );
 		LUA->SetField( -2, "VersionNum" );
 
 		LUA->SetField( GarrysMod::Lua::INDEX_GLOBAL, "sourcenet" );

--- a/source/server/datapack.cpp
+++ b/source/server/datapack.cpp
@@ -121,7 +121,7 @@ namespace DataPack
 #if defined SYSTEM_WINDOWS
 
 	const char GModDataPackProxy::SendFileToClient_sig[] =
-		"\x55\x8B\xEC\x83\xEC\x30\x56\x57\x8B\x7D\x0C\x8B\xF1\x85\xFF\x0F\x88";
+		"\x55\x8B\xEC\x83\xEC\x40\x53\x57\x8B\x7D\x0C";
 	const size_t GModDataPackProxy::SendFileToClient_siglen =
 		sizeof( GModDataPackProxy::SendFileToClient_sig ) - 1;
 

